packages/micropip/test_micropip.py

```diff
@@ -508,7 +511,9 @@ async def test_list_wheel_package(
 
 
 @pytest.mark.asyncio
-async def test_list_wheel_name_mismatch(mock_fetch: mock_fetch_cls, mock_importlib):
+async def test_list_wheel_name_mismatch(
+    mock_fetch: mock_fetch_cls, mock_importlib: None
+) -> None:
```

Comment:
I'm a bit dubious about the usefulness of typing the return type in all test functions, but I imagine without it mypy would complain with the strict typing options?

Reply:
I have mostly not added typing to anything yet. I’m only finishing off the typing for things that were partially typed. I would probably avoid requiring tests to be fully typed. The cost to benefit ratio is low.

Reply:
https://twitter.com/henryschreiner3/status/1527091252709183492?s=21&t=yeEMFASJ1TZml-hCG2CoWA

Reply:
It's useful to put `mock_fetch_cls` in because then language servers know what attributes `mock_fetch` will have.

Reply:
Types are not harmful in tests, just not really worth the effort to add. They still improve readability (especially for fixtures), and IDE's benefit from them. But if you add one type, it's good to go ahead and add all of them, otherwise they are just implicitly Any anyway.